### PR TITLE
Chronos : add [how to tune a forecaster model] notebook

### PR DESCRIPTION
**Pull request by rnwang04**
## Description

Add how-to-tune-a-forecaster-by-tune guide to our documentation for better user experience. 

### 1. How to test?
- [x] Document test
https://ruonantetdoc.readthedocs.io/en/add_how_to_tune_tutorial/doc/Chronos/Howto/index.html
https://ruonantetdoc.readthedocs.io/en/add_how_to_tune_tutorial/doc/Chronos/Howto/how_to_tune_forecaster_model.html

### CODE REVIEW

**Comment by TheaperDeng** [comment]
We will make all the how-to guides executable in colab. This one might be a good example: https://bigdl.readthedocs.io/en/latest/doc/Chronos/Howto/how_to_train_forecaster_on_one_node.html

**Comment by TheaperDeng** [comment]
# how to tune a forecaster through `Forecaster.tune`

Introduction

In this guidance, we demonstrate **how to tune forecaster on single node**. In tuning process, forecaster will find the best hyperparameter combination among user-defined search space, which is a common process if users pursue a forecaster with higher accuracy.

Chronos support forecasting model's hyperparameter tuning in 2 sepaerated APIs (i.e. `Forecaster.tune` and `AutoTSEstimator`) for  users with different demands:

|                   | Single Node | Cluster | performance-awared tuning | feature selection | customized model |
| ----------------- | ----------- | ------- | ------------------------- | ----------------- | ---------------- |
| `Forecaster.tune` | ✅           | ❌       | ✅                         | ❌                 | ❌                |
| `AutoTSEstimator` | ✅           | ✅       | ❌                         | ✅                 | ✅                |

`Forecaster.tune` provides easier and more stright-forward API for users who are familiar with Chronos forecasters, it is recommened to try this method first.

We will take `AutoformerForecaster` and nyc_taxi dataset as an example in this guide.

**Comment by TheaperDeng** [comment]
1. `!pip install --pre --upgrade bigdl-chronos[pytorch,automl]`, I think there should not be a space between the comma
  
2. for Data preparation, we don't need to mention what formats is supported by `fit`.
  
3. In[1], we may make the code as simple as possible, if we won't use test_data, then we should not create it.
  
4. We don't need to discribe the data's shape, it's not the thing we want our users to focus on in this how-to guide.
  
5. discribe some **critical** parameters that users always need to set e.g. target, direction... and leave other parameter to default value. If some parameter is not **critical** and you still need to set this parameter, then change their default values.
  
6. change `newformer` to `new_autoformer`
  
7. provide an "open in colab" button
  
8. teach users how to set AutoformerForecaster's init parameter in save and load

**Review by TheaperDeng — APPROVED** [review]
LGTM

**Review by TheaperDeng — COMMENTED** [review]
1. open in colab is not effective on readthedoc, https://ruonantetdoc.readthedocs.io/en/add_how_to_tune_tutorial/doc/Chronos/Howto/how_to_tune_forecaster_model.html
2. the table is not shown on readthedoc (optional)
3. introduction -> Introduction
4. forcaster -> forecaster https://ruonantetdoc.readthedocs.io/en/add_how_to_tune_tutorial/doc/Chronos/Howto/index.html
